cmake/Modules/FindAzureSDK_EP.cmake

```diff
@@ -83,38 +85,88 @@ if (NOT AZURESDK_FOUND)
     endif()
 
     if (WIN32)
-      set(CFLAGS_DEF "${CMAKE_C_FLAGS}")
-      set(CXXFLAGS_DEF "${CMAKE_CXX_FLAGS}")
+      if(MSVC)
+        set(CFLAGS_DEF " -I${TILEDB_EP_INSTALL_PREFIX}/include /Dazure_storage_lite_EXPORTS /DCURL_STATICLIB=1 ${CMAKE_C_FLAGS}")
```

Comment:
@ihnorton I don't think we can conditionally set `CURL_STATICLIB=1` can we? If we find curl on the system it might be a shared library, so this needs to switch based on the detected curl library from `FindCurl_EP`. Do you agree?

Reply:
Yes, agree

Reply:
Need to remove `/DCURL_STATICLIB` here and below.